app/views/ways_to_help/index.html.erb

```diff
@@ -0,0 +1,161 @@
+      </article>
+
+      <article class="ways_options -has-list">
+        <article>
+          <h3>Procure ajuda</h3>
+          <p>Os aspectos psicológicos têm sido fortemente afetados nesse momento de pandemia: ansiedade com o isolamento e incertezas com o que estar por vir são as ocorrências mais comuns. Se ofereça a ajudar, caso você tenha background para isso e evite ficar fornecendo só notícias ruins.</p>
+          <p>Caso você esteja precisando de ajuda psicológica, considere procurar um profissional. Mesmo que não tenha condições financeiras, algumas instituições estão oferecendo ajuda de maneira gratuita nesse momento.</p>
+          <p><strong>Não deixe de se cuidar!</strong> Alguns exemplos:</p>
+        </article>
+        <section class="list list_ways">
+          <ul>
+            <li class="list_row">
+              <ul class="list_columns">
+                <li><strong>Gerar Arteterapia e Bem Estar - Escuta Virtual Solidária</strong></li>
+                <li>Durante a quarentena COVID-19 - Gratuita para todas as pessoas a partir de 18 anos.</li>
+                <li data-th="Site"><a class="-small" target="_blank" href="https://www.pesolidario.com.br" target="_blank">Pernambuco Solidário contra o Coronavírus</a></li>
+              </ul>
+            </li>
+            <li class="list_row">
+              <ul class="list_columns">
+                <li><strong>Como vai você?</strong></li>
+                <li>Distanciamento da família e perda da sensação de utilidades agravam o isolamento social dos idosos. Converse com quem você confia e acredita que pode te ajudar.</li>
+                <li data-th="Telefone/Instagram">
+                  <a href="tel: 188">Ligue 188</a>
+                  <a class="-small" target="_blank" href="https://www.instagram.com/cvvoficial/" target="_blank">Instagram do Centro de Valorização da Vida</a>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </section>
+      </article>
+
+      <article class="ways_options">
+        <h3>Se puder, Fique em casa</h3>
+        <p>Seguir as diretrizes da saúde pública da sua região, apesar de obrigação, já é uma enorme ajuda. Entretanto, <strong>lembre-se que para a covid-19 você pode ser um vetor de transmissão</strong>, então, o máximo que você puder ficar em casa, melhor para desacelerar a transmissão da doença.</p>
+        <p>Fique em casa o máximo possível e, quando precisar sair, pratique as regras de distanciamento social e de higiene.</p>
+      </article>
+
+      <article class="ways_options">
+        <h3>Cuide de você</h3>
+        <p>Faça esforço para se manter saudável, para isso se alimente bem, descanse e evite atividades de alto risco. Assim, além de reduzir suas chances para contrair o coronavírus, você ajuda a evitar outras doenças e oprimir os sistemas de saúde.</p>
```

Comment:
I would change this sentence to:

"Se mantenha saudável. Para isso, se alimente bem, beba muita água, descanse e evite atividades de alto risco. Assim, você evita outras doenças e acidentes que podem te levar a utilizar o sistema de saúde."